elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/dsl/QueryDsl.scala

```diff
@@ -63,34 +64,12 @@ trait QueryDsl extends DslCommons with SortDsl {
     def apply(query: Query,
```

Comment:
This to supply default of None of optional args. I guess we can do that in the class interface without apply. Updating